scripts/ci/check_identity.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
```

Comment:
have we decided if we are making things generic for python2 or 3, or only doing python3?